core/src/vmware/vmware_cbt_tool/vmware_cbt_tool.py

```diff
@@ -136,14 +144,14 @@ def main():
     # CentOS/RHEL since 7.4 by default do SSL cert verification,
     # we then try to disable it here.
     # see https://github.com/vmware/pyvmomi/issues/212
-    py_ver = sys.version_info[0:3]
-    if py_ver[0] == 2 and py_ver[1] == 7 and py_ver[2] >= 5:
-        import ssl
-
-        try:
-            ssl._create_default_https_context = ssl._create_unverified_context
-        except AttributeError:
-            pass
+    #    py_ver = sys.version_info[0:3]
```

Comment:
Do we want to remove these comments?

Reply:
Indeed, the comment lines 143-154 can be removed. This is now handled by using SmartConnectNoSSL from pyvmomi to skip SSL verification after issuing a warning when SSL verification fails.

https://github.com/bareos/bareos/blob/5548b1544433178d1b86023cf9ec61b18dc0df51/core/src/vmware/vmware_cbt_tool/vmware_cbt_tool.py#L143-L154